docs/DeveloperGuide.adoc

```diff
@@ -543,42 +546,16 @@ The following activity diagram summarizes what happens when a user executes a `*
 .Activity Diagram for `task add` Command
 image::TaskAddActivityDiagram.png[]
 
-===== Searching tasks by date
-The search task feature allows users to search all tasks that occur on the specified date, month, or year.
-This feature is facilitated by `TaskCommandParser`, `TaskSearchCommandParser` and `
-TaskSearchCommand`.
-The operation is exposed in the `Model` interface as `Model#updateFilteredTaskList()`.
-
-Given below is an example usage scenario and how the `*task* search` mechanism behaves at each step:
-
-1. The user executes the `*task* search` command and provides the day, month, or year, or any combination of which
-that they want to search for search for.
-2. `TaskSearchCommandParser` creates a new `TaskSearchCommand` based on the names.
-3. `LogicManager` executes the `TaskSearchCommand`.
-4. `ModelManager` updates the `filteredTasks` in `ModelManager`.
-
-The following sequence diagram shows how the `*task* search` command works:
-
-.Sequence Diagram for `*task* search` Command
-image::TaskSearchSequenceDiagram.png[]
-
-NOTE: The lifeline for `TaskCommandParser`, `TaskSearchCommandParser`, `TaskSearchCommand` and `TaskSearch` should end at
-the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of the diagram.
-
-The following activity diagram summarizes what happens when a user executes a task find command:
-
-.Activity Diagram for `*task* search` Command
-image::TaskSearchActivityDiagram.png[]
-
 //tag::task-edit[]
 ===== Editing a task
-The `task edit` command allows user to edit a task in `ModManager`. The fields that can be edited are: description and time.
+The `task edit` command allows user to edit a task in Mod Manager.
+
```

Comment:
Why change to less specific?

Reply:
I resolved the merge conflict on GitHub, there was 200+ lines so maybe I lost it somewhere